api/proto/info/info.proto

```diff
@@ -212,10 +212,29 @@ message ZInfoSW {
 
 // Errors in response to the application of configuration
 message ErrorInfo {
-  string description = 1;
-  google.protobuf.Timestamp timestamp = 2;
+  string description = 1;                   // Description of the error
+  google.protobuf.Timestamp timestamp = 2;  // Timestamp at which error had occurred
+  Severity severity = 3;                    // Severity of the error
+  DeviceEntity entity = 4;                  // device entity which error belongs to
+  string entityId = 5;                      // uuid of the device entity
```

Comment:
The suggestion from Sahil was to have a sequence of <DeviceEntity, uuid> so that we can show waiting on multiple objects.

Also, we should have an explicit retry_condition string so we can have text for the cases we do not yet have an entity to refer to, such as the time based retry of a download.

Reply:
Also, yetus will complain about the camelcase for entityId; spell it as entity_id and it will be happy.
protoc will generate a go struct where the field is called EntityId in that case.

Reply:
updated as per the input, please check.